packages/core/src/App/Components/Elements/Modals/RedirectNotice/redirect-notice-modal.jsx

```diff
@@ -53,20 +53,18 @@ const RedirectNoticeModal = ({ is_logged_in, is_eu, portal_id }) => {
     }, [is_logged_in, is_eu]);
 
     return ReactDOM.createPortal(
-        dialog_status ? (
-            <Dialog
-                className='redirect-notice'
-                is_visible={dialog_status}
-                title='Redirect notice'
-                is_open={dialog_status}
-                cancel_button_text={localize('Cancel')}
-                confirm_button_text={localize('Proceed')}
-                onCancel={onCancelDialog}
-                onConfirm={onConfirmDialog}
-            >
-                {localize('You are being redirected to an external website.')}
-            </Dialog>
-        ) : null,
+        <Dialog
+            className='redirect-notice'
+            is_visible={dialog_status}
+            title='Redirect notice'
+            cancel_button_text={localize('Cancel')}
+            confirm_button_text={localize('Proceed')}
+            onCancel={onCancelDialog}
+            onConfirm={onConfirmDialog}
+            dismissable={onCancelDialog}
```

Comment:
@aizad-deriv it looks like dismissable accepts a string. please check🙂

Reply:
@maryia-deriv `dismissable` accepts a boolean. `onCancelDialog` is a function that returns `true` or `false` which I have passed the same prop  as `onCancel`.